pages/guides/3rdparty/koreader.mdx

```diff
@@ -90,3 +90,12 @@ With this, you can now push/pull reading progress to/from Kavita.
 
 <Callout type="warning" emoji="⚠️">The automatic sync can potentially undo reading progress. We **strongly** advise against using it. We also cannot support you on recovering potentially lost progress.</Callout>
 
+#### Archive (CBR/CBZ files) Streaming
+
+We do not have KOReader Sync Server setup with archives at this time, KOReader does however make use of the OPDS-PS standard which Kavita supports.
```

Comment:
It would be good to link against the 2 FRs (Archive and PDF) so we can get a good understanding of importance to community as well.